MonoAnalyzerPhoton/interface/MonoCuts.h

```diff
@@ -281,7 +313,7 @@ class MonoCuts:public MonoCandidate, public Photon
   int dEdXOnly_count=0;
   int QualdEdX_count=0;
 
-  // for trigger study
+  // for trigÏr study
```

Comment:
?

Reply:
I'm not sure. Probably some typo with the keyboard.